### DISCUSSION TITLE
Feature Request: Obsidian Support

### DISCUSSION THREAD

**Discussion by NukaCody** [code-hike/codehike#346] [Ideas and feedback]
Is it possible to create an obsidian plugin with Code Hike?

https://obsidian.md/

**Comment by mrboen94** [comment]
As much as I wanted this myself when writing up documents, I am not sure if this is the correct focus for this repo at the moment. Thats just my 5 cents though. I was looking into making my own plugin for this an make a PR for it.

**Comment by pomber** [comment]
I have no experience with obsidian plugins. I'll move this to a discussion just in case someone wants to give it a try.

**Comment by yulei-chen** [comment]
I made one: https://github.com/yuleicul/obsidian-mdx
